public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,253 @@
+openapi: 3.0.0
+info:
+  title: Check a UK EORI Number
+  description: |
+    # Overview

```

Comment:
I think it was mentioned in the documentation that we have to remove the overview section as it is now under common/overview.md? I may be wrong :) can you please double check?

Reply:
please ignore it

Reply:
I see the documentation here https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=DTRG&title=RAML+to+OpenAPI+Specification+%28OAS%29+migration+guide mentions removing the Overview section, yet the example it links to has it in here https://github.com/hmrc/api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml. I'm not sure what is correct - what are your thoughts?

Reply:
As i said please ignore my comment :)